### PR TITLE
Addon useStories -> CSF

### PR DESCRIPTION
**Pull request by dvnrsn**
Issue:

## What I did

Update a section in Addon docs to use CSF rather than `useStories`.
Looking to do the rest of this page, just wanted to make sure I'm thinking about this clearly.

## How to test

- Is this testable with Jest or Chromatic screenshots?
- Does this need a new example in the kitchen sink apps?
- Does this need an update to the documentation?

If your answer is yes to any of these, please make sure to include it in your PR.

### CODE REVIEW

File: docs/src/pages/addons/introduction/index.md [hunk 1]
```diff
@@ -43,6 +43,34 @@ export const defaultView = () => (
...
+CSF
+```js
+import React from 'react';
+import { action } from '@storybook/addon-actions';
+import Button from './button';
+
+const styles = {
+  textAlign: 'center',
+};
+const CenterDecorator = ;
```
Comment (dvnrsn) [inline docs/src/pages/addons/introduction/index.md]: remove this line

**Comment by shilman** [comment]
Thanks for the PR! I think the example you labeled as `storiesOf` is already written in `CSF` though... :)

**Comment by dvnrsn** [comment]
> Thanks for the PR! I think the example you labeled as `storiesOf` is already written in `CSF` though... :)

It is. I was hesitant to remove `storiesOf`. Will proceed to remove as I believe that is probably what is intended.

**Comment by dvnrsn** [comment]
Looks like @ndelangen beat me to it... closing

**Comment by ndelangen** [comment]
Ow now, sorry for that @dvnrsn, your contributions are very welcome, If I had known you were working on this, I would have left it to you.

Again, thank you so much for your time and efforts 🙇

**Pull request closed without merging**